src/_data/nav.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
       pageKey: 'play',
       pageSection: 'start_a_club',
     }, {
-      label: 'Verbinde mit der Community',
+      label: 'Sag »Hallo!«',
```

Comment:
This will need additional glyphs in the font for the quotation marks - see #6 for related issue